proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -225,6 +225,17 @@ message TimerTaskInfo {
     bytes branch_token = 12;
 }
 
+
+message ArchivalTaskInfo {
```

Comment:
is this added in https://github.com/temporalio/temporal/pull/3524?

Reply:
Yes, see this comment in the PR description:

> This PR depends on https://github.com/temporalio/temporal/pull/3524, so there's an extra commit here. PTAL at the latest commit on this branch